core/cache/cachetest/controller.go

```diff
@@ -17,6 +17,9 @@ import (
 // TestController wraps a cache controller for testing.
 // It allows synchronisation of state objects with the cache
 // without the need for a multi-watcher and cache worker.
+// This is useful when testing with StateSuite;
+// JujuConnSuite sets up a cache worker and multiwatcher
+// to keep the model cache in sync.
```

Comment:
Is this comment still valid?
You sounded like you had taken a path that you changed your mind, but I don't see where 'controller.go' is doing this.

Reply:
I have clarified this comment to indicate that is is not necessary when using `JujuConnSuite`.